docs/bin/bt

```diff
@@ -9,7 +9,7 @@
 #
 
 require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile',
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../../Gemfile',
```

Comment:
you see i got the one in `bin/bridgetown` it might be worth making this configurable in future Bridgetown releases.